src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -234,7 +238,22 @@ public ComputeEngineInstance provision(TaskListener listener, Label requiredLabe
             Instance i = instance();
             Operation operation = cloud.client.insertInstance(cloud.projectId, i);
             logger.println("Sent insert request");
-            ComputeEngineInstance instance = new ComputeEngineInstance(cloud.name, i.getName(), i.getZone(), i.getDescription(), runAsUser, "./.jenkins-slave", numExecutors, mode, requiredLabel == null ? "" : requiredLabel.getName(), new ComputeEngineLinuxLauncher(cloud.getCloudName(), operation, this.useInternalAddress), new CloudRetentionStrategy(retentionTimeMinutes), getLaunchTimeoutMillis());
+            ComputeEngineComputerLauncher launcher = new ComputeEngineLinuxLauncher(cloud.getCloudName(), operation, this.useInternalAddress);
```

Comment:
This will need to be reconciled with the WindowsVM support being added in PR#10. Perhaps abstracting out the launcher initialization into a helper method which can handle both cases.

Reply:
I will see when merging what to do with that..

Reply:
After merge change here is even smaller that it was.. and there is no problem with windows..